.github/workflows/add all issues to project

```diff
@@ -0,0 +1,18 @@
+name: Add issues to bugs project
+
+on:
+  issues:
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@RELEASE_VERSION
+        with:
+          project-url: https://github.com/orgs/saving-satoshi/projects/2
+          github-token: ${{ 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 }}
```

Comment:
@ecurrencyhodler You should consider placing this token in github secrets. Github recommends treating the personal access token as sensitive data https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.

Reply:
This secret should probably get rotated

Reply:
Yeah. I just got notified that the PAT was revoked from github. I'm not sure how to make this work. I initially hoped fine grained PAT's would work but sounds like they don't:
https://github.com/actions/add-to-project/issues/289

Reply:
@0xBEEFCAF3 if I put PAT into a github secret, will this work?

Reply:
yup. Put it in GitHub secrets and then refer to it using `${{ secrets.SECRET_TOKEN }}`

Reply:
@0xBEEFCAF3 Like this? https://github.com/saving-satoshi/saving-satoshi/pull/101